test/cmd.js

```diff
@@ -74,7 +74,8 @@ describe('express(1)', function () {
         '    "http-errors": "~1.6.3",\n' +
         '    "jade": "~1.11.0",\n' +
         '    "morgan": "~1.9.1"\n' +
-        '  }\n' +
+        '  },\n' +
+        '  "devDependencies": {}\n' +
```

Comment:
It is probably not add this if there are none; `npm` will end up removing it on the next `npm I` command a user uses. That's why we sort them in the same way npm does as well, so it doesn't cause churn in the file after the fact.

Reply:
Got it, will remove it and only add devDependencies on TypeScript templates.

```js
pkg.devDependencies = {}
pkg.devDependencies['typescript'] = '~3.7.5'
// ...rest of codes
```